Tests/SwiftGodotMacrosTests/MacroGodotExportCategoryTests.swift

```diff
@@ -0,0 +1,1449 @@
+//
+//  MacroGodotExportGroupTests.swift
+//  SwiftGodotMacrosTests
+//
+//  Created by Estevan Hernandez on 12/4/23.
+//
+
+import SwiftSyntaxMacros
+import SwiftSyntaxMacrosTestSupport
+import XCTest
+import SwiftGodotMacroLibrary
+
+final class MacroGodotExportGroupTests: XCTestCase {
+    let testMacros: [String: Macro.Type] = [
+        "Godot": GodotMacro.self,
+        "Export": GodotExport.self,
```

Comment:
this macro may not need to be included, because technically the exportGroup macro is only operative in the Godot macro